steps/set-ios-product-bundle-identifier/1.0.0/step.yml

```diff
@@ -0,0 +1,46 @@
+title: Replace PRODUCT_BUNDLE_IDENTIFIER
+summary: Replaces Xcode's project PRODUCT_BUNDLE_IDENTIFIER setting.
+description: |-
+  Replaces Xcode's project `PRODUCT_BUNDLE_IDENTIFIER` setting by passing in a new value.
+  This is the recommended way to substitute the bundle identifier instead of modifiying the `CFBundleIdentifier` setting in the `Info.plist` file.
+
+  From [Xcode 7 Release Notes](https://developer.apple.com/library/content/documentation/Xcode/Conceptual/RN-Xcode-Archive/Chapters/xc7_release_notes.html):
+  > The new build setting Product Bundle Identifier (PRODUCT_BUNDLE_IDENTIFIER) is the recommended place to set the Bundle Identifier for a target. The target’s Info.plist should be configured to use this build setting by referencing it as $(PRODUCT_BUNDLE_IDENTIFIER) in the value for the CFBundleIdentifier key.
+website: https://github.com/emenegro/bitrise-step-product-bundle-identifier
+source_code_url: https://github.com/emenegro/bitrise-step-product-bundle-identifier
+support_url: https://github.com/emenegro/bitrise-step-product-bundle-identifier/issues
+published_at: 2017-03-28T11:05:05.50225827+02:00
+source:
+  git: https://github.com/emenegro/bitrise-step-product-bundle-identifier.git
+  commit: f218dde9c751806bc952183d682005d072012207
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- script
+- build
+- utils
+- xcode
+- xamarin
```

Comment:
type tags require a bit of normalization - @godrei can you please review & advise?